components/backends/vllm/src/dynamo/vllm/main.py

```diff
@@ -208,7 +201,7 @@ async def init(runtime: DistributedRuntime, config: Config):
         config, factory
     )
 
-    # TODO Hack to get data, move this to registering in ETCD
+    # TODO Hack to get data, move this to registering in shared storage somewhere
```

Comment:
"shared storage somewhere" 🤣

maybe "the registry"?

Reply:
Humm, but "the registry" doesn't exist. I changed it to "TBD" until we figure it out.